### PR TITLE
Development

### PR DESCRIPTION
**Pull request by dasari-ananya**

### CODE REVIEW

File: src/utility/Image.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//       alert("Image ratio" + img.width / img.height);
+//       resolve({ width, height });
+//     };
+//     img.onerror = function(error) {
+//       reject(error);
+//     };
+//   });
+//   const imgSrc = "https://cdn.pixabay.com/photo/2015/03/26/09/47/sky-690293_1280.jpg";
+//   return { width: 3, height: 1 };
+// }
```
Comment (ksridharbabuus) [inline src/utility/Image.js]: Do we need this file, if we dont need you can delete it. Thanks!

File: package.json [hunk 2]
```diff
@@ -88,22 +91,22 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.6.0",
-    "@storybook/addon-actions": "^5.1.11",
-    "@storybook/addon-links": "^5.1.11",
-    "@storybook/addons": "^5.1.11",
-    "@storybook/react": "^5.1.11",
+    "@babel/core": "^7.12.10",
```
Comment (ksridharbabuus) [inline package.json:94]: I see there are many packages got upgraded. Can you please make sure that the functionality is in tact with the latest versions across these packages.

**Review by ksridharbabuus — COMMENTED** [review]
Provide couple of comments, please look into it.

File: src/utility/Image.js (hunk 1 above)
Comment (dasari-ananya) [reply src/utility/Image.js]: deleted the file

**Review by ksridharbabuus — APPROVED** [review]
Approved.

**Pull request merged**